backend/telemetry_core/src/aggregator/aggregator.rs

```diff
@@ -44,6 +44,8 @@ pub struct AggregatorOpts {
     /// How many nodes from third party chains are allowed to connect
     /// before we prevent connections from them.
     pub max_third_party_nodes: usize,
+    /// Flag to expose the node's ip address to the feed subscribers.
+    pub expose_ip: bool,
```

Comment:
Nit: I'd call this something like "expose_node_ips" to try and make it clearer whay it does ("expose_ip" offhand could sound like it's about exposing the up of the telemetry service in some way)

Reply:
Yep, I like that name better